circus-train-integration-tests/pom.xml

```diff
@@ -14,9 +15,20 @@
     <hadoop.version>2.8.1</hadoop.version>
     <!-- override for hadoop-common dependency that is used in this test -->
     <guava.version>17.0</guava.version>
+    <surefire.excluded.groups>com.hotels.bdp.circustrain.integration.utils.SchemaEvolution</surefire.excluded.groups>
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+      <version>2.5</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+      <version>2.5</version>
```

Comment:
Minor, but since you have this twice and they're related you could make this a property.